src/index.html

```diff
@@ -8,6 +8,90 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <h1 class="heading">Stars</h1>
+
+    <div class="stars__container">
+      <ul class="stars stars--0">
+        <li class="stars__star"></li>
+
+        <li class="stars__star"></li>
+
+        <li class="stars__star"></li>
+
+        <li class="stars__star"></li>
+
+        <li class="stars__star"></li>
```

Comment:
They are pretty much the same, so I would say that spaces are redundant here. It does not improve readability, however, makes your code less ergonomic
```suggestion
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
```